web/modules/custom/portland/portland.module

```diff
@@ -256,7 +256,7 @@ function portland_preprocess_menu(&$vars)
           unset($vars['items'][$key]);
           return;
         }
-        $item['title'] = Drupal\Core\Render\Markup::create('Alerts <span class="badge badge-dark">' . $alertCount . '</span>');
+        $item['title'] = Drupal\Core\Render\Markup::create('Alerts <span class="badge bg-dark">' . $alertCount . '</span>');
```

Comment:
You're right, I think it may be a relic from the past :)

![image](https://user-images.githubusercontent.com/1520776/217345403-340c1f25-b4d4-4312-a774-571dae3e0550.png)

I can remove this and related code.